### PR TITLE
feat: Speakers: rank column added

### PR DESCRIPTION
**Pull request by MeghalBisht**
Reqd for : fossasia/open-event-frontend#5443

#### Short description of what this resolves:
rank column added

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

File: migrations/versions/rev-2021-01-06-23:25:53-e3f69cc43643_.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+# revision identifiers, used by Alembic.
+revision = 'e3f69cc43643'
+down_revision = '2d0760003a8a'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('sessions_speakers_links', 'session_state')
```
Comment (iamareebjamal) [inline migrations/versions/rev-2021-01-06-23:25:53-e3f69cc43643_.py:21]: Why

File: migrations/versions/rev-2021-01-06-23:25:53-e3f69cc43643_.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    op.add_column('speaker', sa.Column('rank', sa.Integer(), nullable=False))
```
Comment (iamareebjamal) [inline migrations/versions/rev-2021-01-06-23:25:53-e3f69cc43643_.py:22]: server default missing. Did you try to run the migrations?

Comment (MeghalBisht) [reply migrations/versions/rev-2021-01-06-23:25:53-e3f69cc43643_.py:22]: Yes, I don't know why it's missing.

Comment (iamareebjamal) [reply migrations/versions/rev-2021-01-06-23:25:53-e3f69cc43643_.py:22]: You have to manually add it

File: app/api/schema/speakers.py [hunk 3]
```diff
@@ -53,7 +53,7 @@ class Meta:
     country = fields.Str(allow_none=True)
     city = fields.Str(allow_none=True)
     gender = fields.Str(allow_none=True)
-    rank = fields.Integer(allow_none=True, default=0)
+    rank = fields.Integer(allow_none=True, server_default='0')
```
Comment (iamareebjamal) [inline app/api/schema/speakers.py:56]: default=0

File: app/models/speaker.py [hunk 4]
```diff
@@ -33,7 +33,7 @@ class Speaker(SoftDeletionModel):
     country = db.Column(db.String)
     city = db.Column(db.String)
     gender = db.Column(db.String)
-    rank = db.Column(db.Integer, default=0, nullable=False)
+    rank = db.Column(db.Integer, server_default='0', nullable=False)
```
Comment (iamareebjamal) [inline app/models/speaker.py:36]: Where is default. server_default is not a valid sqlalchemy kwarg

Comment (MeghalBisht) [reply app/models/speaker.py]: Please check now

**Pull request merged**